src/typed-method-types/workflows/triggers/webhook_test.ts

```diff
@@ -0,0 +1,40 @@
+import { assertEquals } from "https://deno.land/std@0.99.0/testing/asserts.ts";
+import { WebhookTrigger } from "./webhook.ts";
+import { TriggerTypes } from "./mod.ts";
+
+Deno.test("Webhook Triggers can set the type using the string", () => {
+  const webhook: WebhookTrigger = {
+    type: "webhook",
+    name: "test",
+    workflow: "example",
+    inputs: {},
+  };
+  assertEquals(webhook.type, TriggerTypes.Webhook);
+});
+
+Deno.test("Webhook Triggers can set the type using the TriggerTypes object", () => {
+  const webhook: WebhookTrigger = {
+    type: TriggerTypes.Webhook,
+    name: "test",
+    workflow: "example",
+    inputs: {},
+  };
+  assertEquals(webhook.type, TriggerTypes.Webhook);
+});
+
+Deno.test("Webhook Trigger Filters support an optional webhook object", () => {
+  const webhook: WebhookTrigger = {
```

Comment:
Could we manually test creating a "real" filter with a variable binding confirm they work as expected end-to-end?

Reply:
Okay, I updated the Reacji Channeler example and the following is working e2e! I'll share a message in channel to point you to the new branch
```ts
    await client.workflows.triggers.create({
      type: "event",
      name: "Reacji Channeler",
      workflow: "#/workflows/reacji_channeler",
      event: {
        event_type: "slack#/events/reaction_added",
        channel_ids: [listen_channel],
        filter: {
          version: 1,
          root: {
            statement: `{{data.reaction}} == ${reaction}`,
          },
        },
      },
      inputs: {
        // use trigger context data for the message channel/ts
        message_channel: { value: "{{data.channel_id}}" },
        message_ts: { value: "{{data.message_ts}}" },
        forward_channel: { value: forward_channel },
      },
    });
```